charts/csm-application-mobility/values.yaml

```diff
@@ -50,32 +62,36 @@ velero:
 # Based on the objectstore being used , the velero plugin and its configuration may need to change! Below is the sample configuration for using aws object store.
 # GCP and Azure plugins configuration are different. See more details at: https://github.com/vmware-tanzu/helm-charts/blob/main/charts/velero/README.md
   configuration:
-    provider: aws
     backupStorageLocation:
-      name: default
-      bucket: velero-bucket
-      config: {}
-      #  region:
-      #  s3ForcePathStyle:
-      #  s3Url:
-      #  kmsKeyId:
-      #  resourceGroup:
-      #  The ID of the subscription containing the storage account, if different from the cluster’s subscription. (Azure only)
-      #  subscriptionId:
-      #  storageAccount:
-      #  publicUrl:
-      #  Name of the GCP service account to use for this backup storage location. Specify the
-      #  service account here if you want to use workload identity instead of providing the key file.(GCP only)
-      #  serviceAccount:
+      - name: default
```

Comment:
Documentation table (https://dell.github.io/csm-docs/docs/applicationmobility/deployment/#configuration) will need to be updated since we now take a slice of backupStorageLocations.